backend/src/graphql/arg/SearchUsersFilters.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, InputType, ObjectType } from 'type-graphql'
+
+@ObjectType()
+@InputType('SearchUsersFiltersInput')
```

Comment:
```suggestion
@InputType('SearchUsersFilters')
```

Reply:
same same 👆🏼
This is the reason why I had to use the argument here, because I got a duplicated type names error. @Mogge